css/style.css

```diff
@@ -0,0 +1,150 @@
+#top {
+  background-color: var(--color-red);
+}
+
+img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+p {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.footer-social p {
+  font-size: 12px;
+}
+
+.footer-ul {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.footer-logo {
+  object-fit: unset;
+}
+
+#top img {
+  object-fit: unset;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.card a {
+  font-size: 24px;
+  font-weight: 700;
+}
+
+.debate a {
+  font-size: 17px;
+  line-height: 21px;
+  font-weight: 500;
+}
+
+.magazine a {
+  line-height: 20px;
+}
+
+a:hover {
+  color: black;
+  text-decoration-color: #f72210;
+}
+
+#top ul li a:hover {
+  color: var(--color-figcaption);
+}
+
+.title {
+  color: var(--color-grey);
+  font-weight: bold;
+  font-size: 14px;
+}
+
+figure figcaption {
+  background: var(--bg-figcaption);
+  color: var(--color-figcaption);
+  bottom: 0;
+  left: 0;
+  font-size: large;
+  font-weight: 800;
+}
+
+.card figcaption {
+  font-size: 12px;
+}
+
+.debate span {
+  font-size: 14px;
+  font-family: Roboto, "Open Sans", "Alata", sans-serif;
+}
+
+.debate article:first-child {
+  border-bottom: 4px solid var(--color-grey);
+}
+
+.debate article:nth-child(2) {
+  border-bottom: 4px solid var(--color-grey);
+}
```

Comment:
Group selectors with the same style definition

Reply:
ok thanks, it's done